### DISCUSSION TITLE
tls-ca-bundle.pem origin, mounted in filebeat

### DISCUSSION THREAD

**Discussion by stijnpieters** [Security-Onion-Solutions/securityonion#1997] [Unsupported Versions] [answered]
Dumb question or not, I found a certificate that I don't know where it is from, so I'll ask anyway.
There's a certificate, stored on my manager in /opt/so/conf/ca/tls-ca-bundle.pem
It's from some Venezuelan CA
It's mounted in my filebeat container

Does this have any actual function, or is it there by accident?

Removed some parts because it already takes up enough space:
Certificate:
    Data:
        Version: 3 (0x2)
    Signature Algorithm: sha1WithRSAEncryption
        Issuer: CN=ACCVRAIZ1, OU=PKIACCV, O=ACCV, C=ES
        Validity
            Not Before: May  5 09:37:37 2011 GMT
            Not After : Dec 31 09:37:37 2030 GMT
        Subject: CN=ACCVRAIZ1, OU=PKIACCV, O=ACCV, C=ES
        Subject Public Key Info:
            Public Key Algorithm: rsaEncryption
                Public-Key: (4096 bit)
                Exponent: 65537 (0x10001)
        X509v3 extensions:
            Authority Information Access: 
                CA Issuers - URI:http://www.accv.es/fileadmin/Archivos/certificados/raizaccv1.crt
                OCSP - URI:http://ocsp.accv.es
            X509v3 Subject Key Identifier: 
                D2:87:B4:E3:DF:37:27:93:55:F6:56:EA:81:E5:36:CC:8C:1E:3F:BD
            X509v3 Basic Constraints: critical
                CA:TRUE
            X509v3 Authority Key Identifier: 
                keyid:D2:87:B4:E3:DF:37:27:93:55:F6:56:EA:81:E5:36:CC:8C:1E:3F:BD
            X509v3 Certificate Policies: 
                Policy: X509v3 Any Policy
                  User Notice:
                    Explicit Text: 
                  CPS: http://www.accv.es/legislacion_c.htm
            X509v3 CRL Distribution Points: 
                Full Name:
                  URI:http://www.accv.es/fileadmin/Archivos/certificados/raizaccv1_der.crl
            X509v3 Key Usage: critical
                Certificate Sign, CRL Sign
            X509v3 Subject Alternative Name: 
                email:accv@accv.es

[tls-ca-bundle.pem.txt](https://github.com/Security-Onion-Solutions/securityonion/files/5568037/tls-ca-bundle.pem.txt)

**Comment by TOoSmOotH** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/blob/dev/salt/elasticsearch/files/scripts/so-catrust

It's pulling the ca.crt and adding the internal CA to the docker container so that TLS works properly for the TLS plugin that encrypts Elastic traffic.
